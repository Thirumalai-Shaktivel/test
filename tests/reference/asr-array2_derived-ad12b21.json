{
    "basename": "asr-array2_derived-ad12b21",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/array2_derived.f90",
    "infile_hash": "e369ff08e97bdc709b67c7ba894c208aebac6f07f89726c76925733c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-array2_derived-ad12b21.stdout",
    "stdout_hash": "8816745bdf866012c7c55e829ee43d234889f382db88e2a4498ec6c2",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}