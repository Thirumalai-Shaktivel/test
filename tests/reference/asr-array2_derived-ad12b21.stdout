(TranslationUnit (SymbolTable 1 {array2_derived: (Program (SymbolTable 2 {a: (Variable 2 a Local () Default (Real 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 5 (Integer 4 [])))]) Source Public Required), b: (Variable 2 b Local () Default (Real 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 5 (Integer 4 [])))]) Source Public Required), c: (Variable 2 c Local () Default (Integer 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 3 (Integer 4 [])))]) Source Public Required), d: (Variable 2 d Local () Default (Logical 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 2 (Integer 4 [])))]) Source Public Required), der_type: (DerivedType (SymbolTable 3 {i: (Variable 3 i Local () Default (Integer 4 []) Source Public Required), r: (Variable 3 r Local () Default (Real 4 []) Source Public Required)}) der_type Source Public), e: (Variable 2 e Local () Default (Real 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 2 (Integer 4 []))) ((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 3 (Integer 4 [])))]) Source Public Required), f: (Variable 2 f Local () Default (Integer 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 3 (Integer 4 []))) ((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 4 (Integer 4 [])))]) Source Public Required), g: (Variable 2 g Local () Default (Logical 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 5 (Integer 4 []))) ((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 2 (Integer 4 [])))]) Source Public Required), h: (Variable 2 h Local () Default (Real 4 [((ConstantInteger 2 (Integer 4 [])) (ConstantInteger 3 (Integer 4 []))) ((ConstantInteger 3 (Integer 4 [])) (ConstantInteger 4 (Integer 4 []))) ((ConstantInteger 4 (Integer 4 [])) (ConstantInteger 5 (Integer 4 [])))]) Source Public Required), i: (Variable 2 i Local () Default (Integer 4 [((ConstantInteger 3 (Integer 4 [])) (ConstantInteger 4 (Integer 4 []))) ((ConstantInteger 4 (Integer 4 [])) (ConstantInteger 5 (Integer 4 []))) ((ConstantInteger 3 (Integer 4 [])) (ConstantInteger 4 (Integer 4 [])))]) Source Public Required), j: (Variable 2 j Local () Default (Logical 4 [((ConstantInteger 5 (Integer 4 [])) (ConstantInteger 6 (Integer 4 []))) ((ConstantInteger 2 (Integer 4 [])) (ConstantInteger 3 (Integer 4 []))) ((ConstantInteger 2 (Integer 4 [])) (ConstantInteger 4 (Integer 4 [])))]) Source Public Required), p: (Variable 2 p Local () Default (Derived 2 der_type [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 7 (Integer 4 [])))]) Source Public Required), q: (Variable 2 q Local () Default (Derived 2 der_type [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 7 (Integer 4 []))) ((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 9 (Integer 4 [])))]) Source Public Required), r: (Variable 2 r Local () Default (Derived 2 der_type [((ConstantInteger 7 (Integer 4 [])) (ConstantInteger 10 (Integer 4 []))) ((ConstantInteger 9 (Integer 4 [])) (ConstantInteger 15 (Integer 4 []))) ((ConstantInteger 16 (Integer 4 [])) (ConstantInteger 18 (Integer 4 [])))]) Source Public Required), x: (Variable 2 x Local () Default (Complex 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 6 (Integer 4 [])))]) Source Public Required), y: (Variable 2 y Local () Default (Complex 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 6 (Integer 4 []))) ((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 8 (Integer 4 [])))]) Source Public Required), z: (Variable 2 z Local () Default (Complex 4 [((ConstantInteger 6 (Integer 4 [])) (ConstantInteger 7 (Integer 4 []))) ((ConstantInteger 8 (Integer 4 [])) (ConstantInteger 10 (Integer 4 []))) ((ConstantInteger 11 (Integer 4 [])) (ConstantInteger 13 (Integer 4 [])))]) Source Public Required)}) array2_derived [] [])}) [])
